src/router/routes.ts

```diff
@@ -38,39 +38,84 @@ export default (store: Store<State>): RouteRecordRaw[] => {
     },
     {
       path: '/zones',
-      name: 'zones',
-      meta: {
-        title: 'Zones',
-      },
-      props: (route) => ({
-        selectedZoneName: route.query.zone,
-        offset: getLastNumberParameter(route.query.offset),
-      }),
-      component: () => import('@/app/zones/views/ZonesView.vue'),
+      children: [
+        {
+          path: '',
+          name: 'zone-list-view',
+          meta: {
+            title: 'Zone',
```

Comment:
Should this be `Zones`?

Reply:
Yes, indeed. I’ll change that in my follow-up PR as that deals mostly with the list views anyway.

Reply:
👍